site/content/installation-upgrade/installation-oss.md

```diff
@@ -292,6 +292,42 @@ Before you install NGINX Agent for the first time on your system, you need to se
     sudo nginx-agent -v
     ```
 
+### Installing NGINX Agent on Amazon Linux 2023
+
+1. Install the prerequisites:
+
+    ```shell
+    sudo dnf install yum-utils procps
```

Comment:
not sure if yum-utils required with dnf. also procps name can be procps-ng - needs checking